app/classes/document_search_results.rb

```diff
@@ -5,7 +5,7 @@ def initialize(result, offset = 0)
     @total = result['hits']['total']
     @offset = offset
     @results = extract_hits(result['hits']['hits'])
-    @suggestion = extract_suggestion(result['suggest']['suggestion']) if result['suggest']
+    @suggestion = extract_suggestion(result['suggest'])
   end
```

Comment:
did you still want to check result['suggest'] ?

Reply:
Yes, that holds the suggestion info:
```ruby
{
    "suggest" => {
        :suggestion => [
            [0] {
                   :text => "fsands",
                 :offset => 0,
                 :length => 6,
                :options => [
                    [0] {
                               :text => "funds",
                        :highlighted => "funds",
                              :score => 0.0066609583
                    }
                ]
            }
        ]
    }
}
```

Reply:
The logic to check if a suggestion is present has been moved to the `extract_suggestion` method:
```ruby
def extract_suggestion(suggest)
    return unless suggest...
```